exporter/splunkhecexporter/README.md

```diff
@@ -28,7 +28,11 @@ The following configuration options can also be configured:
 - `max_content_length_logs` (default: 2097152): Maximum log data size in bytes per HTTP post limited to 2097152 bytes (2 MiB).
 - `max_content_length_metrics` (default: 2097152): Maximum metric data size in bytes per HTTP post limited to 2097152 bytes (2 MiB).
 - `splunk_app_name` (default: "OpenTelemetry Collector Contrib") App name is used to track telemetry information for Splunk App's using HEC by App name.
-- `splunk_app_version` (default: Current OpenTelemetry Collector Contrib Build Version): App version is used to track telemetry information for Splunk App's using HEC by App version.
+- `splunk_app_version` (default: Current OpenTelemetry Collector Contrib Build Version): App version is used to track telemetry information for Splunk App's using HEC by App version. 
+- `allow_log_data` (default: true): Specifies whether the log data is exported. Set it to `false` if you want the log 
```

Comment:
I think that logs_enabled and profiling_enabled might be better here

Reply:
Yes, this is what I've been think of as well. My reasoning for `allow` over `enabled` is that enabled can be interpreted in many ways e.g. whether this exporter expose its own logs or profiling data, while `allow` is more like _forward the data or drop it_. But I don't have a strong opinion here. Maybe @aunshc can chime in as well

Reply:
I tend to think enabled is better, though also consider disabled since default true properties can be a bit confusing

Reply:
Ok, I updated it to `log_data_enabled` and `profiling_data_enabled`. I used the `data` to ovoid confusion with `logs` pipeline data type and something like exporter's own logs and profiling. @emaderer @anuraaga let me know what do you think?